torch/onnx/symbolic_opset13.py

```diff
@@ -109,6 +109,136 @@ def unsafe_split_with_sizes(g, self, split_sizes, dim, _outputs=None):
+
+            # Loop inputs
+            loop_block = utils._add_block(loop.node())
+            block_input_iter = utils._add_input_to_block(loop_block)
+            cond = utils._add_input_to_block(loop_block)
+            final_splits = utils._add_input_to_block(loop_block)
+
+            start = loop_block.op(
+                "Gather", indices_or_sections, block_input_iter, axis_i=0
+            )
+            end = loop_block.op(
+                "Gather",
+                indices_or_sections,
+                loop_block.op("Add", block_input_iter, const_1),
+                axis_i=0,
+            )
+
+            slice = loop_block.op("Slice", self, start, end, axis)
+            final_splits = loop_block.op("SequenceInsert", final_splits, slice)
+
+            # Loop outputs
+            cond_out = loop_block.op("Cast", loop_condition, to_i=9)
+            utils._add_output_to_block(loop_block, cond_out)
+            utils._add_output_to_block(loop_block, final_splits)
+
+            loop_out = loop.node().output()
+            start = g.op(
+                "Gather",
+                indices_or_sections,
+                g.op("Constant", value_t=torch.tensor(-1, dtype=torch.long)),
+                axis_i=0,
+            )
+            start = opset11.unsqueeze(g, start, 0)
+            end = symbolic_helper._size_helper(g, self, axis)
+
+            last_slice = g.op("Slice", self, start, end, axis)
+
+            return g.op("SequenceInsert", loop_out, last_slice)
+
+        else:  # scalar tensor
+            dim_size = symbolic_helper._size_helper(g, self, axis)
+            min_split_size = g.op("Div", dim_size, indices_or_sections)
+            min_split_size_plus_1 = g.op(
+                "Add",
+                min_split_size,
+                g.op("Constant", value_t=torch.tensor(1, dtype=torch.long)),
+            )
+            num_splits_one_extra = g.op("Mod", dim_size, indices_or_sections)
+            splits = g.op("Tile", min_split_size_plus_1, num_splits_one_extra)
+            leftover = g.op(
+                "Tile",
+                min_split_size,
+                g.op(
+                    "Sub",
+                    opset11.unsqueeze(g, indices_or_sections, 0),
+                    num_splits_one_extra,
+                ),
+            )
+
+            splits = g.op("Concat", splits, leftover, axis_i=0)
+            if _outputs is None:
+                return g.op("SplitToSequence", self, splits, axis_i=dim)
+            return g.op("Split", self, splits, axis_i=dim, outputs=_outputs)
+
+    split_val = indices_or_sections.node()["value"]
+    if split_val.dim() > 0:
+        return static_tensor_split(self, indices_or_sections)
+    split_size = symbolic_helper._get_const(
+        indices_or_sections, "i", "indices_or_sections"
+    )
+
+    size = symbolic_helper._get_tensor_dim_size(self, dim)
+    if size is None:
+        if _outputs is not None:
```

Comment:
It doesn't make sense for `_outputs == 0`, right?
If so, then make this more pythonic with:
```suggestion
        if _outputs:
```

Reply:
Nearly all of the other `_outputs` checks in `symbolic_opsetX.py` files compare with an integer or `None`, so I think we just leave it as is (maybe more explicit).

Reply:
OK, makes sense to keep it consistent for now.